frontend/src/components/dashboard/subdomain/SubdomainInfoTooltip.module.scss

```diff
@@ -0,0 +1,73 @@
+@import "../../../styles/tokens.scss";
+@import "~@mozilla-protocol/core/protocol/css/includes/lib";
+@import "~@mozilla-protocol/core/protocol/css/includes/mixins/typography";
+
+.info-icon {
+  align-items: center;
+  background-color: transparent;
+  border-style: none;
+  color: $color-light-gray-70;
+  cursor: pointer;
+  display: flex;
+}
+
+.explainer-wrapper {
+  background-color: $color-white;
+  box-shadow: $box-shadow-sm;
+  border-radius: $border-radius-md;
+  padding: $spacing-lg;
+  // On small screens, appear as a modal at the top of the viewport:
+  top: $spacing-lg;
+  left: $spacing-lg;
+  right: $spacing-lg;
+  position: absolute;
+
+  &::before {
+    content: "";
+    height: 12px;
+    width: 12px;
+    position: absolute;
+    top: -6px;
+    left: 12px;
+    transform: rotate(45deg);
+    background-color: $color-white;
+    display: none;
+  }
+
+  @media screen and (min-width: 970px) {
```

Comment:
suggestion (blocking): use media query tokens

Reply:
I tried using the tokens, but there was nothing between screen-md and screen-lg. 970px was used mainly because of the max-width used in reference to the figma file, https://github.com/mozilla/fx-private-relay/pull/3712#discussion_r1277737089